src/modules/i18n/locales/de.json

```diff
@@ -658,5 +658,7 @@
     "operation_successful": "Erfolg!",
     "operation_failed": "Hoppla! Etwas ist schiefgegangen. Bitte versuchen Sie es erneut.",
     "operation_aborted": "Operation abgebrochen",
-    "outside_valid_year_range": "Das eingegebene Jahr liegt außerhalb des verfügbaren Bereichs:"
+    "outside_valid_year_range": "Das eingegebene Jahr liegt außerhalb des verfügbaren Bereichs:",
+    "linkedin_tooltip": "Teilen Sie diese Karte auf LinkedIn",
+    "share_map_title": "Link zum Geoportal der Schweizerischen Eidgenossenschaft"
```

Comment:
@stebie @hansmannj @mariokeusen for the share by email button, on windows with outlook it seems that we need to have a subject otherwise the body is set in the subject as follow: `&body=https://s.geo.admin.ch/asdfasdf`. That's why I created a generic email subject for the sharing, could you review it (in german and english) ? Or do you have a better proposal as default email subject ?